assessment/report_generator.go

```diff
@@ -66,6 +67,100 @@ func GenerateReport(dbName string, assessmentOutput utils.AssessmentOutput) {
 	logger.Log.Info("completed publishing sample report")
 }
 
+func writeNonSchemaChanges(dbName string, nonSchemaChanges [][]string) {
+	f, err := os.Create(dbName + "_non_schema_changes.txt")
+	if err != nil {
+		logger.Log.Error(fmt.Sprintf("Can't create schema file %s: %v", dbName, err))
```

Comment:
Please correct the error message - this should probably non schema report

Reply:
Done.